data/sources/manifests/mci/0203.manifest.yaml

```diff
@@ -0,0 +1,27 @@
+data_version: 0.0.1
+description: MCI Description
+groups:
+  mci-0203:
```

Comment:
So to guarantee order, should the group names be made a list (as the blocks are)?